src/pages/Book/index.js

```diff
@@ -4,35 +4,81 @@ import Cover from "../../components/Cover";
 import Title from "../../components/Title";
 import Author from "../../components/Author";
 import Description from "../../components/Description";
-import { Container, CoverDiv } from "./book.styles";
+import Loading from "../../components/Loading";
+import { Container, CoverDiv, Favorite } from "./book.styles";
+import { FaStar, FaRegStar } from "react-icons/fa";
 
 class Book extends React.Component {
   state = {
     data: null,
     isLoading: false,
   };
 
-  getBook = async (isbn) => {
+  getBook = async (id) => {
     try {
       this.setState({ isLoading: true });
       const { data } = await axios(
-        `https://www.googleapis.com/books/v1/volumes?q=isbn:${isbn}&key=${process.env.REACT_APP_API_KEY}`
+        `https://www.googleapis.com/books/v1/volumes/${id}`
       );
-      this.setState({ data: data.items[0], isLoading: false });
+      this.setState({ data: data, isLoading: false });
```

Comment:
this will be overwritten by the next set state and you cause a double render, what is the purpose of this?

Reply:
Just a mistake. I've corrected it. Thanks for catching it.